packages/lesswrong/components/posts/PostsPage/PostsPageActions.jsx

```diff
@@ -22,26 +26,50 @@ const showPostActions = (currentUser, post) => {
     Posts.canEdit(currentUser, post)
 }
 
-const PostsPageActions = ({classes, post, setMenuAnchor, anchorEl, currentUser}) => {
-  if (showPostActions(currentUser, post)) {
-    return <span>
-      <MoreHorizIcon className={classes.icon} onClick={e => setMenuAnchor(e.target)}/>
-      <Menu
-        anchorEl={anchorEl}
-        open={Boolean(anchorEl)}
-        onClose={() => setMenuAnchor(null)}
-      >
-        <Components.PostActions Container={MenuItem} post={post}/>
-      </Menu>
-    </span>
-  } else {
-    return null
+class PostsPageActions extends PureComponent {
+  state = { anchorEl: null }
+
+  handleClick = (e) => {
+    if (this.state.anchorEl) {
```

Comment:
Seems like we can reduce this whole if statement to a one-liner with a tertiary condition